src/containers/NotePop/NotePop.test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { ErrorPop, mapStateToProps } from './NotePop';
+import { shallow } from 'enzyme';
+
+describe('ErrorPop', () => {
+  let wrapper;
+  beforeEach(() => {
+    wrapper = shallow(
+      <ErrorPop/>
+    )
+  });
+
+  it('should match the snapshot', () => {
+    expect(wrapper).toMatchSnapshot()
+  });
+
+  describe('mapStateToProps', () => {
+
+    it('should return a string', () => {
+      const mockState = {error: 'Title is Required'}; 
```

Comment:
@DevelopingDavid On MSTP testing you should have more properties than you are returning for the component in your mockState and test that you are only returning the properties you need in the component. Look at the MSTP test in App.test.js for reference.